src/app/technical-record/dda/dda.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, OnInit, ChangeDetectionStrategy, Input } from '@angular/core';
+import { DDA } from '@app/models/tech-record.model';
+
+@Component({
+  selector: 'vtm-dda',
+  templateUrl: './dda.component.html',
+  styleUrls: ['../../app.component.scss'],
```

Comment:
we need to remove this line

Reply:
nice one for spotting it, I don't know why we load this file referenced in app, we agreed we will be using _global.scss (if and only if new component styles are required for components that do not exist) and no scoped css